MitigationApi.yml

```diff
@@ -5,7 +5,7 @@ info:
   description: 'Often faults can be detected and resolved by a system or person and this information should be passed on to the user to indicate that the action is available (and whether to take that action or not)'
   contact:
     email: g.dorchies@resilienx.com
-  version: 0.0.8
+  version: 0.0.9

```

Comment:
Still needs a changelog update.

Reply:
would this be a minor update instead of a patch?

Reply:
I don't think this quite meets the criteria for it to be a minor over a patch, its just a new field for an already existing type and comparatively there have been many similar changes to this and other API's that were also just a patch

Reply:
Once 1.0.0 has been hit @AS-resilienx , yes, this would be minor, but we anticipate a lot of flux with this API short term, so it is a little bit of 'wild west' rules for now.

Reply:
Ok, understandable! Thanks!